src/main/resources/application.yaml

```
@@ -16,7 +16,16 @@ spring:
   config:
     import: "optional:configtree:/mnt/secrets/"
   application:
-    name: Spring Boot Template
+    name: HMC HMI Inbound Adapter
+  jackson:
+    deserialization:
+      fail-on-unknown-properties: true
+    serialization:
+      WRITE_DATES_AS_TIMESTAMPS: false
+
+hmc:
+  cft-hearing-service:
+    host: http://localhost:4459
```

Can we define an env var for this too please?
```suggestion
    host: ${CFT_HEARING_SERVICE_URL:http://localhost:4459}
```